opt_maven_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -x
+
+# $1=OPENSHIFT_CI=true means running in CI
+if [[ "$1" == "true" ]]; then
+
+  yum -y install --setopt=skip_missing_names_on_install=False \
+      curl \
+      java-1.8.0-openjdk-devel \
+      java-1.8.0-openjdk \
+      protobuf protobuf-compiler \
```

Comment:
@jupierce if you already messed with the protobuf we're getting here with no luck, did you also try changing the protobuf-compiler (protoc). Running protoc can be somewhat OS dependent though it's not meant to be -- it is. Maybe we can tell yum which version, maybe not the latest, of protoc to use to try to get a happy Maven build?